pkg/repository/backend_postgres_migrations/019_add_workspace_multi_gpu.go

```diff
@@ -0,0 +1,21 @@
+package backend_postgres_migrations
+
+import (
+	"database/sql"
+
+	"github.com/pressly/goose/v3"
+)
+
+func init() {
+	goose.AddMigration(upAddFieldWorkspaceMultiGpuEnabled, downDropFieldWorkspaceMultiGpuEnabled)
```

Comment:
I think this is deprecated. Can we use the AddMigrationContext function instead?